Slime Time: Minor midround slimeperson customization!

mcmeiler opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Lets Jellypeople choose their mutant color, hair style, facial hair style, if applicable, ears and tails through an action button. Also makes slimepeople able to wag tails if they have one.

This is my first "actual" PR, feel free to complain about my shitcode.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Jellypeople needed something more unique about them, currently the only unique things are that they have no blood loss and reverse toxin damage.
Besides, cosmetic options are good, and not all ships have a mirror.

(Note, this action lets you go above the limit of how dark mutant colors can go, however this was approved by Zeta.)
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added midround Jellyperson customization! Included are: Mutant Color, hair style, facial hair style, ears and tails. (More may follow.)
imageadd: Sprites for slime cat ears and tails
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

mcmeiler committed:
e

mcmeiler committed:
actually fixes it

thanks to ivanmixo for finding it

mcmeiler committed:
Removes restriction on dark mutation colors

Per request from Zeta

mcmeiler committed:
Removes ear selection for now

Might come back to it later when I got the nerve for it, it's very late.

mcmeiler committed:
Removes an unused var

ZephyrTFA commented on code/modules/mob/living/carbon/human/species_types/jellypeople.dm:
```diff
@@ -24,15 +25,18 @@
-	if(regenerate_limbs)
+	if(regenerate_limbs || humanoid_customization)
 		regenerate_limbs.Remove(C)
+		humanoid_customization.Remove(C)
```
```suggestion
	if(regenerate_limbs)
		regenerate_limbs.Remove(C)
	if(humanoid_customization)
		humanoid_customization.Remove(C)
```

mcmeiler replied:
Honestly should've thought of that myself, then again, I'm not a full time coder and mostly did this because I felt like doing it.

mcmeiler committed:
Suggestion as per review

Co-authored-by: Zephyr <12817816+ZephyrTFA@users.noreply.github.com>

mcmeiler committed:
Re-adds ear swap functionality

Took like 5 minutes lol

mcmeiler committed:
Merge branch 'basedslimetime' of https://github.com/mcmeiler/Shiptest into basedslimetime

mcmeiler committed:
Re-does ear code, fixes an issue with ear selection

mcmeiler committed:
Hooo boy I'm not proud of this

Summary:
Added slime cat ears, exclusive to slimepeople's alter shape action

Added low-alpha sprite for slime cat ears

mcmeiler committed:
Adds tail alteration and tail wagging for jellies

mcmeiler commented:
I might add more things to this eventually, however that will probably be in another PR, this was more or less intended as the framework for more.

mcmeiler commented:
I think lints is bullshitting me like bruh

``
Run yogstation13/DreamAnnotate@v1
Error: ENOENT: no such file or directory, open 'output-annotations.txt'
``
EDIT: No I'm just filter-worded lol

triplezeta commented:
linter says your indentation is fucked

mcmeiler committed:
I want to eat my spacebar

mcmeiler commented:
should be fixed in commit
